bin/cml/runner.js

```diff
@@ -397,15 +390,22 @@ const run = async (opts) => {
     );
     if (availableRunner) {
       winston.info('Found matching idle runner.', availableRunner);
-      process.exit(0);
+      return;
     }
   }
 
-  if (driver === 'github') {
+  if (dockerVolumes.length && cml.driver !== 'gitlab')
+    winston.warn('Parameters --docker-volumes is only supported in gitlab');
+
+  if (driver === 'github')
     winston.warn(
       'Github Actions timeout has been updated from 72h to 35 days. Update your workflow accordingly to be able to restart it automatically.'
     );
-  }
+
+  if (RUNNER_NAME)
+    winston.warn(
+      'ignoring RUNNER_NAME environment variable, use CML_RUNNER_NAME or --name instead'
+    );
```

Comment:
off-topic, but maybe its time to remove this warning?

Reply:
Might be! There is indeed a collection of vars used in the original containers when we launched the runner via docker.
They might be reviewed

Reply:
But not here. We should open another issue